.github/workflows/tics-tox.yaml

```diff
@@ -12,6 +12,11 @@ on:
         required: false
         default: '.'
         type: string
+      fetch_libs:
+        description: 'Whether to fetch the latest charm libs'
```

Comment:
Not necessarily latest. In our case it will fetch the versions specified in `charmcraft.yaml`

Reply:
Good point!